recipe/meta.yaml

```diff
@@ -50,6 +50,8 @@ source:
     - patches/0002-specify_libadd.patch
     - patches/0003-install_netcdfcxxconfig_cmake.patch
     - patches/0004-fix_CHECK_LIBRARY_EXISTS.patch
+    - patches/0005-normalize-eols-examples.patch
+    - patches/0006-Use_HAVE_NC_SET_LOG_LEVEL.patch
```

Comment:
Ugh, this patching was messy!  There are some weird end-of-line characters that made this much harder than it seems like it should have been.